### PR TITLE
Update haar_cascade.py

### PR DESCRIPTION
**Pull request by tatsuyaokunaga**
line 149 & 197
old) flags=cv2.cv.CV_HAAR_SCALE_IMAGE 
new) flags=cv2.CASCADE_SCALE_IMAGE

### CODE REVIEW

**Comment by kamathhrishi** [comment]
@tatsuyaokunaga Hello , Could you clarify more on what your pull request does?

**Comment by tatsuyaokunaga** [comment]
@kamathhrishi Hello,
As written on this page, https://github.com/mpatacchiola/deepgaze/issues/41
I got below error,
AttributeError: module 'cv2' has no attribute 'CV_HAAR_SCALE_IMAGE'

Such an error occurs because the submodule cv has been removed from opencv 3.0.
You can use an alternative cv2.CASCADE_SCALE_IMAGE.

File: deepgaze/haar_cascade.py [hunk 1]
```diff
@@ -146,7 +146,7 @@ def _findFrontalFace(self, inputImg, scaleFactor=1.1, minSizeX=30, minSizeY=30,
             scaleFactor=scaleFactor,
             minNeighbors=minNeighbors,
             minSize=(minSizeX, minSizeY),
-            flags=cv2.cv.CV_HAAR_SCALE_IMAGE
+            flags=cv2.CASCADE_SCALE_IMAGE
```
Comment (kamathhrishi) [inline deepgaze/haar_cascade.py:149]: Oh , awesome. Thank you for the fix 👍 

@mpatacchiola will merge it when he logs in.

**Review by kamathhrishi — APPROVED** [review]

File: deepgaze/haar_cascade.py (hunk 1 above)
Comment (mpatacchiola) [reply deepgaze/haar_cascade.py:149]: I think this should be handled explicitly at some point in the pipeline. 
I mean, we could check if the version of OpenCV is 2 or 3 and use the correct flag.

However, recall that the master branch of Deepgaze is currently compatible only with OpenCV 2, meaning that this patch cannot be merged on the master. We should instead check if it has been correctly implemented on the branch 2.0 which is compatible with OpenCV 3.

Comment (kamathhrishi) [reply deepgaze/haar_cascade.py:149]: Oh, correct. @mpatacchiola We have to still port haar cascades to 2.0. I think we could add this explicit fix for 2 or 3. The thing is that I have seen certain parts of the codebase work with 3.0 already.

Comment (mpatacchiola) [reply deepgaze/haar_cascade.py:149]: This has now been added to branch 2.0

**Comment by mpatacchiola** [comment]
This has now been added in branch 2.0

**Pull request closed without merging**